lib/spriteBackgroundImages.js

```diff
@@ -1,11 +1,10 @@
 const queryString = require('querystring');
 const { promisify } = require('util');
 const packers = require('./packers');
-let Canvas;
+const canvasPrebuilt = require('canvas-prebuilt');
 
-try {
-    Canvas = require('canvas');
-} catch (e) {}
+const Canvas = canvasPrebuilt.Canvas;
```

Comment:
Could use destructuring?

Reply:
Sure. It's that ok with our node version support?

Reply:
Destructuring assignment is node 6.5.0+ so yeah. We should also lebab/prettierify the code a bit :)

Reply:
Getting ready to introduce prettier etc. over here: https://github.com/assetgraph/assetgraph-sprite/pull/182